model/patch/patch.go

```diff
@@ -1273,15 +1273,15 @@ func GetCollectiveStatusFromPatchStatuses(statuses []string) string {
 	}
 
 	if hasCreated && (hasFailure || hasSuccess) {
-		return evergreen.PatchStarted
+		return evergreen.VersionStarted
 	} else if hasCreated {
-		return evergreen.PatchCreated
+		return evergreen.VersionCreated
 	} else if hasFailure {
-		return evergreen.PatchFailed
+		return evergreen.VersionFailed
 	} else if hasAborted {
-		return evergreen.PatchAborted
+		return evergreen.VersionAborted
 	} else if hasSuccess {
-		return evergreen.PatchSucceeded
+		return evergreen.LegacyPatchSucceeded
```

Comment:
Do we want to return VersionSucceeded here instead?

Reply:
I'd like to stick with returning the legacy value right now, in case this value is written to the DB anywhere. Writing with the new status is what I'd like to address in the next PR.

Reply:
SGTM